platforms/kops_aws.env

```diff
@@ -8,7 +8,7 @@ KUBE_SSH_USER=admin
 LOG_DUMP_SSH_KEY=/workspace/.ssh/kube_aws_rsa
 LOG_DUMP_SSH_USER=admin
 LOG_DUMP_SAVE_LOGS=cloud-init-output
-LOG_DUMP_SAVE_SERVICES=protokube
+LOG_DUMP_SAVE_SERVICES="protokube kops-configuration"
```

Comment:
I don't think you need the quote here.. humm seems unit test does not check platforms, need to add the coverage